src/tracks/SampleTrack.cpp

```diff
@@ -309,11 +309,8 @@ void SampleTCO::loadSettings( const QDomElement & _this )
 	if( _this.hasAttribute( "color" ) )
 	{
 		useStyleColor( false );
-		useCustomClipColor( _this.attribute( "isclipcolor" ).toInt() );
-		
-		QColor loadedColor;
-		loadedColor.setNamedColor( _this.attribute( "color" ) );
-		setColor( loadedColor );
+		useCustomClipColor( _this.attribute( "isclipcolor" ) == "1" ? true : false );
```

Comment:
`condition ? true : false` is equivalent to `condition`. Same applies to three other instances of the same line (in AutomationPattern.cpp, Pattern.cpp, and BBTrack.cpp)

```suggestion
		useCustomClipColor( _this.attribute( "isclipcolor" ) == "1");
```